net-libs/libwebsockets/libwebsockets-4.0.1-r1.ebuild

```diff
@@ -32,8 +31,9 @@ RDEPEND="
 	libuv? ( dev-libs/libuv:= )
 	sqlite3? ( dev-db/sqlite:= )
 	ssl? (
-		!libressl? ( dev-libs/openssl:0= )
-		libressl?  ( dev-libs/libressl:0= )
+		ssl_libressl?  ( dev-libs/libressl:0= )
+		ssl_mbedtls? ( >net-libs/mbedtls-2.0.0 )
+		ssl_openssl? ( dev-libs/openssl:0= )
```

Comment:
Adjust as per previous comments:

ssl? (
   libressl? ( ... )
   mbedtls? ( ... )
   !libressl? ( !mbedtls? ( ... ))
)

Just check syntax for the last line please.